src/util.js

```diff
@@ -0,0 +1,19 @@
+/**
+   Public util functions for the SDK.
+ */
+
+/**
+   Constructs an image variant definition query param. Does not validate what
+   param names or values, as that is left for the API.
+*/
+// eslint-disable-next-line import/prefer-default-export
+export const imageVariantURLStr = opts => {
```

Comment:
I wonder if this should accept "width" & "height" as alternative to "w" & "h"? Or would that be too confusing? 
Was there some decision regarding that?

Reply:
My thinking was that as they are not the actual endpoint parameters and can't be found in the API reference, they'll be confusing. Then again, probably people are more likely to read FTW code than the API reference.